src/actions/account-actions.js

```diff
@@ -21,8 +21,12 @@ export const createSession = (googleOauthJWT) => {
 
   return (dispatch) => {
     axios(requestConfig)
-      .then(response => dispatch({ type: CREATE_SESSION, payload: response.data.user_info }))
-      .catch(error => dispatch({ type: NETWORK_ERROR, payload: error }));
+      .then((response) => {
```

Comment:
@KingPra These were changes made during debugging that you probably want to drop.